evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -203,25 +214,10 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
                 findActive(2).css({transform: "", opacity: ""});
             }
 
-            handle("repeat-unreviewed");
+            repeat("unreviewed");
+            updateButtons();
```

Comment:
To `repeat`, the call to `updateButtons` can be integrated, whereas in `slide` not